doc/sqa_fenix.yml

```diff
@@ -2,6 +2,23 @@ directories:
     - ${ROOT_DIR}/test/tests
 specs:
     - tests
-dependencies:
+dependencies: # commented-out categories are not available
     - framework
+    #- chemical_reactions
+    - electromagnetics
+    - fluid_properties
+    - heat_transfer
+    - misc
+    - navier_stokes
+    - phase_field
+    - reactor
+    - rdg
+    - ray_tracing
+    - scalar_transport
+    - solid_mechanics
+    - solid_properties
+    - stochastic_tools
+    - thermal_hydraulics
+    - tmap8
+    #- cardinal
```

Comment:
Again, I'm wondering why cardinal is commented out.

Reply:
Again, Cardinal doesn't use the SQA system. This is there as a placeholder if it does (and a reminder to devs that we didn't neglect it).